### PR TITLE
feat: check that module crontab is not empy in mfxxx.status

### LINKED ISSUE #122: In mfxxx.status, add test to verify module crontab is not empty

### PR DESCRIPTION
**Pull request by thebaptiste**
close #122

### CODE REVIEW

File: adm2/templates/mfxxx.status [hunk 1]
```diff
@@ -39,6 +39,21 @@ if test "${ADMIN_HOSTNAME}" != "null"; then
...
+# Checking module crontab is not empty
+echo -n "- Checking crontab is not empty..."
+echo_running
+
+if hash crontab 2>/dev/null; then
+    N=`crontab -l | wc -l`
+    if test ${N} -gt 0; then
+        echo_ok
+    else
+        echo_nok "(crontab is empty)" || RES=1
```
Comment (thefab) [inline adm2/templates/mfxxx.status]: RES=1 must be in next line (without "||")

**Review by thefab — CHANGES_REQUESTED** [review]

**Review by thefab — APPROVED** [review]
